packages/matchbox/index.d.ts

```diff
@@ -0,0 +1,3 @@
+export * from './components';

```

Comment:
This doesn't look like a definitions file, should this be `index.ts`?

Reply:
Oh right so I just ignored the .d.ts files that were generated just to keep the example PR clean to see the main points. It actually generates a .d.ts file for every component / file and exports to `./components` `./hooks` and `./helpers` so we'd need to wrap that all up in one file or one directory we include so FE has the definitions

Reply:
Ohhh i see